test/test-files/messageType/messageType1.js

```diff
@@ -2,6 +2,6 @@
 
 module.exports = {
 	'config' : {
-		'messageType' : 'v1Test'
+		'messageType' : 'v1.test-version.create'
```

Comment:
the new messageTypes were added at the next 'describe' method. [link](https://github.com/shinjiescorido/pathlint/blob/54b82a40190ef8ef161b089a55dfc4bfcd6a4742/test/adapters/message-lint.js#L30)